cmd/destroy.go

```diff
@@ -135,7 +135,9 @@ func destroy(ctx *cli.Context) error {
 			for i, s := range sessions {
 				ss[i] = [3]string{strconv.FormatUint(s.Sid, 10), s.HostName, s.MountPoint}
 			}
-			logger.Fatalf("%d sessions are active, please disconnect them first:\n%s", num, printSessions(ss))
+			logger.Errorf("%d sessions are active, please disconnect them first:", num)
+			fmt.Println(printSessions(ss))
```

Comment:
Why?

Reply:
![image](https://user-images.githubusercontent.com/31313340/188641622-cb71b150-d01d-48e0-bd25-4b269647b193.png)

Reply:
It's not a big deal. After this PR, the list of session will be in STDOUT, but others in STDERR.